front/src/stylesheets/_theming.scss

```diff
@@ -107,9 +107,10 @@ i.zui-subheader-right:hover {
   color: #ffffff;
 }
 
-:not(.mdc-list--non-interactive) > :not(.mdc-list-item--disabled).mdc-list-item::after, :not(.mdc-list--non-interactive) > :not(.mdc-list-item--disabled).mdc-list-item::before {
-  background-color: transparent;
-}
+// :not(.mdc-list--non-interactive) > :not(.mdc-list-item--disabled).mdc-list-item::after,
```

Comment:
Since using git, maybe we can stop commiting commented code.
Still good though, approving